db/src/types.rs

```diff
@@ -38,12 +36,13 @@ pub enum ValueType {
 
 /// Represents a Mentat value in a particular value set.
 // TODO: expand to include :db.type/{instant,url,uuid}.
-#[derive(Clone,Debug,Eq,Hash,Ord,PartialOrd,PartialEq)]
+// Floats and doubles can't implement Eq and Hash.
+#[derive(Clone,Debug,PartialOrd,PartialEq)]
```

Comment:
I'm inclined to expect we will not.  I don't feel strongly about having raw `f64` here.